lib/src/main/java/com/github/kevinsawicki/http/HttpRequest.java

```diff
@@ -287,7 +287,7 @@ public void checkServerTrusted(X509Certificate[] chain, String authType) {
       try {
         SSLContext context = SSLContext.getInstance("TLS");
         context.init(null, trustAllCerts, new SecureRandom());
-        TRUSTED_FACTORY = context.getSocketFactory();
+        TRUSTED_FACTORY = new TLSSocketFactory(context);`
```

Comment:
Ah, crud. Will need to rebuild. My compiler didn't pick this up because I was making the edits in the cached file.